projects/split-view/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@skyux/split-view",
+  "version": "5.0.0-beta.2",
+  "peerDependencies": {
+    "@angular/animations": "^9.1.9 || ^12",
+    "@angular/common": "^12.2.0",
+    "@angular/core": "^12.2.0",
+    "@angular/forms": "^9.1.9 || ^12",
+    "@skyux/core": "^5.0.0-beta.0",
+    "@skyux/i18n": "^5.0.0-beta.0",
+    "@skyux/indicators": "^5.0.0-beta.0"
```

Comment:
These versions should match the version listed in the workspace `package.json`, e.g., `"@skyux/i18n": "^5.0.0-beta.7"`.